src/lib/client.ts

```diff
@@ -1,4 +1,4 @@
-import { Observable, Subject, Subscription } from 'rxjs';
+import { Observable, Subject, Subscription as RxSubscription } from 'rxjs';
```

Comment:
Such a shame the GraphQL folks decided to go with the term `Subscription`.

To avoid conflicts, would it make sense to use the term `SubscriptionQuery` to refer to a GraphQL Subscription?

Reply:
I'm indifferent. Just needs a unique name somehow. I'm fine with SubscriptionQuery and can make that change.

Reply:
Actually. I think it'd be sort of nice to keep the external API using Subscription and renaming rx internally. The reason being that we have types for `Query` `Mutation` and adding a `SubscriptionQuery` would be off convention.

Reply:
Good point - how about `GraphqlQuery` `GraphqlMutation` and `GraphqlSubscription`?

This would prevent anyone creating their own exchanges from having to deal with the same conflict/confusion.

Reply:
I like that direction actually. Good point about the exchange typings. Since the data shapes are an internal implementation, should we name them `UrqlQuery` `UrqlMutation` `UrqlSubscription` ?

Reply:
Even better!

Reply:
I'm a little put off by names that reference the library name 😅 generally that'd indicate some coupling of the API concept and the internals of the library itself. For instance, what if emotion renamed "styled()" to "emotion()"?

The GraphQLMutation naming is good, but the spelling of GraphQL might make it prone to typos?

Reply:
All in all, I'd really go with just `Subscription`, `Query`, and `Mutation` as our Urql component/internals naming. Rx + Urql in a single file is userland and less likely overall to cause naming conflicts. Let's think about what naming makes sense for us first and foremost

Reply:
Okay. Decided. Leaving it as is.